tests/bsim/bluetooth/ll/multiple_id/tests_scripts/multiple_central.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Copyright 2018 Oticon A/S
+# SPDX-License-Identifier: Apache-2.0
+
+source ${ZEPHYR_BASE}/tests/bsim/sh_common.source
+
+# Multiple connection between two devices with multiple peripheral identity
+simulation_id="central_single_peripheral_multilink"
+verbosity_level=2
+EXECUTE_TIMEOUT=1600
+
+cd ${BSIM_OUT_PATH}/bin
+
+Execute ./bs_${BOARD_TS}_tests_bsim_bluetooth_ll_multiple_id_prj_conf \
+  -v=${verbosity_level} -s=${simulation_id} -d=0 -testid=peripheral_multilink
+
+peripheral_count=20
```

Comment:
Suggest to use `peripheral_count` when setting `-D=21` - That way the value can be modified in a single place

Reply:
done